kitsune/sumo/static/sumo/js/wiki.js

```diff
@@ -787,4 +787,20 @@
     });
   }
 
+  var draftbutton = $('.btn-draft');
+  var url = $('.btn-draft' ).data( 'draft-url' );
+  var draftmessage = $('.draft-message').html();
+  draftbutton[0].addEventListener('click', function() {
+    var both_data = $('#both_form').serializeArray();
+    var doc_data = $('#doc_form').serializeArray();
+    var rev_data = $('#rev_form').serializeArray();
```

Comment:
There are 3 type of forms a localizers can see in time by time.
``doc_form`` refers to a form, which contains only document content like title and slug
``rev_form`` refers to a form, which contains revision content like revision content, search summery, and others
``both_form`` refers to a big form, which has both document content and revision content. This form is being showed to the localizers when they localize any article for first time. 
I think the ID name is meaningful as per their use. As we define like the same in the backend views.